packages/react-composites/src/composites/CallWithChatComposite/CallWithChatComposite.tsx

```diff
@@ -177,6 +177,10 @@ const CallWithChatScreen = (props: CallWithChatScreenProps): JSX.Element => {
       setActivePane('none');
     } else {
       setActivePane('chat');
+      // timeout is required to give the window time to render the sendbox so we have something to send focus to.
+      setTimeout(() => {
+        (document.querySelector(`[data-ui-id="sendbox-textfield"]`) as HTMLTextAreaElement)?.focus();
```

Comment:
`data-ui-id` is *strictly* for e2e testing purposes. We should be able to change `data-ui-id` arbitrarily without breaking production code.

Reply:
Okay I will change how it is selected. 

so the major problem here is that the chat pane and people pane are hidden with just CSS right now. this is a issue because the autofocus option is dependent on rendering, so since the chat pane is only rendered once but hidden the autofocus prop does not work

Reply:
So this is a case where we externally want to refocus on the SendBox after initial render.

This would be a good argument to simply implement `.focus()` on `ChatComposite` (and probably `CallWithChatComposite`), where `.focus()` does the right thing:
* For `ChatComposite`, it focuses on the `SendBox` text field
* For `CallWithChatComposite`, it focuses on `MuteButton` if `ChatPane` is hidden, and on `SendBox` if it is shown.

This would be a non-trivial feature though.

Reply:
should we make a user story around this?

Reply:
There's a bug that exists from when we implemented the original autofocus prop: [Bug 2672276: [Web] Contoso cannot call .focus() on composites or components such as sendbox](https://skype.visualstudio.com/SPOOL/_workitems/edit/2672276)